optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/scenario/simplequeens/TestdataQueenEntity.java

```diff
@@ -0,0 +1,41 @@
+package org.optaplanner.core.impl.testdata.domain.scenario.simplequeens;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataQueenDifficultyWeightFactory;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataRowStrengthWeightFactory;
+
+@PlanningEntity(difficultyWeightFactoryClass = TestdataQueenDifficultyWeightFactory.class)
+public class TestdataQueenEntity extends TestdataEntity {
```

Comment:
I am not confortable with putting another copy of the queen class in our repository. That's too much maintenance.
Can we either move these tests into the optaplanner-examples test, so they can reuse the real Queen class. Or can we simply reuse any of the existing testdata classes?